src/action.ts

```diff
@@ -1,9 +1,11 @@
 import * as core from '@actions/core';
+
```

Comment:
**nitpick:** the formatting of the `import` statements is deliberate. Please, revert this change.

Reply:
Done

Reply:
Done